curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fc22d1e64d1b04cdd4e602.md

```diff
@@ -11,22 +11,22 @@ Add some space between the `.inline` elements and the `label` text, by giving a
 
 # --hints--
 
-You should give the `.inline` elements a `margin-right` of `0.5em`.
+You should give the `.inline` elements a `margin-top` of `0`.
```

Comment:
are these changes for this PR?

Reply:
The margin one are quality of life change, the none asked for it and it is unnecessary. I am happy to revert it if asked.